### ISSUE TITLE
Bitbucket organization repositories do not work with `do-not-fork` option

### ISSUE THREAD

**Issue by dpfeiffer** [fthomas/scala-steward#883]
This seems to be related to #689 

Taking a closer look there seems to be a mixup with the source repository when creating the pull request with `do-not-fork` option.

```
{"type": "error", "error": {"fields": {"source": ["repository user/repo not found"]}, "message": "source: repository user/repo not found"}}
```

The proper source would be in this case `organization/repo`.

I hope that I will be able to take a look tomorrow. But either way I am wondering when this was introduced as we had the Scala Steward now running for quite a while without issues. @fthomas do you maybe have a hint, so I don't break anything?

The fix in my opinion could look very similar to the GitLab implementation where we pass the do not fork option into the VCSAlg and switch on the source repo.

**Comment by fthomas** [comment]
@dpfeiffer I guess this has been fixed with the merge of #886. Can we close this?

**Comment by dpfeiffer** [comment]
@fthomas yes we can close it now, it was indeed fixed with #886
